src/main/java/org/openstreetmap/atlas/geography/atlas/pbf/BridgeConfiguredFilter.java

```diff
@@ -52,5 +53,12 @@ public boolean test(final AtlasEntity atlasEntity)
     {
         return this.configuredFilter != null ? this.configuredFilter.test(atlasEntity)
                 : this.configuredTaggableFilter.test(atlasEntity);
+
+    }
+
+    public boolean test(final Taggable taggable)
+    {
+        return this.configuredFilter != null ? this.configuredFilter.test(taggable)
```

Comment:
I am surprised here since `ConfiguredFilter` is a `Predicate<AtlasEntity>` and here you pass it a `Taggable`.

Reply:
Oh you added the `test(Taggable)` method to `ConfiguredFilter` too down there, found it.

Reply:
yeah, it's a bit of a hack but it lets us use `ConfiguredFilter` on the tag maps for entities, which is important at the PbfReader stage since those entities have TagMaps but are not AtlasEntities yet

Reply:
Got it, yes that makes sense!